benchmarks/python/test_batchnorm_bwd.py

```diff
@@ -11,6 +11,9 @@
 @pytest.mark.parametrize("size", generate_input_sizes(dims=4))
 @pytest.mark.parametrize("dtype", FLOAT_DTYPES)
 @pytest.mark.parametrize("channels_last", [True, False])
+@pytest.mark.inner_persistent
+@pytest.mark.outer_persistent
```

Comment:
This should depend on `"channels_last", [True, False]`, is there a way to make marker depends on para?